docs/ako_evh.md

```diff
@@ -1,50 +1,48 @@
 # Enhanced Virtual Hosting support in AKO
 
-This feature supports EVH VS creation from the AKO. `AKOSettings.enableEVH` needs to be set to `true` to enable this feature. Currently, this feature is supported only for Kubernetes Ingress object.
+This feature supports EVH VS creation from the AKO. `AKOSettings.enableEVH` needs to be set to `true` to enable this feature. This feature is supported for Kubernetes Ingress object and Openshift Route object.
 
 ## Overview
 
 AKO currently creates an SNI child VS (Virtual Service) to a parent shared VS for the secure hostname. The SNI VS is used to bind the hostname to a sslkeycert object. The sslkeycert object is used to terminate the secure traffic on Avi's service engine. On the SNI VS, AKO creates httppolicyset rules to route the terminated (insecure) traffic to the appropriate pool object using the host/path specified in the rules section of this ingress object.
 
-With EVH (Enhanced Virtual Hosting) support in AVI, virtual hosting on virtual service can be enabled irrespective of SNI. Also, the SNI can only handle HTTPS (HTTP over SSL) traffic whereas EVH children can handle both HTTP and HTTPS traffic. Unlike SNI which switches only TLS (Transport Layer Security) connections based on one-to-one mapping of children to FQDN (Fully Qualified Domain Name), EVH maps one FQDN to many children based on the resource path requested.
+With EVH (Enhanced Virtual Hosting) support in AVI, virtual hosting on virtual service can be enabled irrespective of SNI. Also, the SNI can only handle HTTPS (HTTP over SSL) traffic whereas EVH children can handle both HTTP and HTTPS traffic. For each unique host in kubernetes/openshift cluster, unique EVH child will be created in secure, insecure deployments. L4 and Passthrough deployments will be similar to that of SNI way of deployment.
 
 With EVH enabled host rule CRD's can be applied to insecure ingress as well. 
 
 More details of EVH can be found here <https://avinetworks.com/docs/20.1/enhanced-virtual-hosting/>.
 
 ### Naming of AVI Objects with EVH enabled
 
+In AVI controller, attributes of AVI objects have size restriction. Name of each AVI object has certain length restriction. AKO generates name of each AVI object by concatenating different fields of ingress/routes/services. Sometime this results in overflow of size restriction imposed by AVI Controller.
```

Comment:
Can we re-word this to:
 Starting Avi Controller 20.1.6, all object names have a max length limitation of 255 characters. To avoid object name lengths beyond 255 characters, AKO will name all EVH object names except the parent virtualservice,  VIP names  and advancedL4 object names using a SHA1 encoding logic.